src/custom-types.json

```diff
@@ -743,6 +743,14 @@
       "https://tools.ietf.org/html/rfc7763"
     ]
   },
+  "text/mdx": {
+    "compressible": false,
```

Comment:
Isn't mdx a text file? This is just a comment, but wanted to not this incase it was a mistake

Reply:
Yeah, not sure what I was thinking, fixed to `true`.